common/Log.cpp

```diff
@@ -119,9 +128,18 @@ namespace Log
                 : _size(0)
                 , _oldest_time_us(0)
             {
+#if ENABLE_DEBUG
```

Comment:
I added this #if because without the assert there is no value in doing atomic operations. It's to minimize performance noise (at runtime), rather than to silence the compiler.

Reply:
Removed.